Core/Controllers/EventsController.cs

```diff
@@ -403,5 +403,9 @@ public IActionResult RemoveVote(string entryDisplayId)
             DatabaseHelpers.Votes.DeleteVote(vote);
             return Ok(new VoteActionResult(true, false));
         }
+
+        private Boolean CanViewHiddenEvent(UserAccount user, Event e) 
```

Comment:
Maybe change `Boolean` to abbreviated `bool` keyword.

Reply:
oh right, this is C#